tools/cldr-code/src/main/java/org/unicode/cldr/tool/ToolConstants.java

```diff
@@ -59,45 +60,64 @@
         "36.1",
         "37.0",
         "38.0",
-        "38.1"
+        "38.1",
+        "39.0"
```

Comment:
should be in a data file, as suggested in CLDR-9149 (perhaps i should dust that off and do it)

Reply:
Yes, good idea.